voltdb/datadog_checks/voltdb/queries.py

```diff
@@ -27,14 +27,14 @@
         {'name': 'memory.rss', 'type': 'gauge'},
         {'name': 'memory.java.used', 'type': 'gauge'},
         {'name': 'memory.java.unused', 'type': 'gauge'},
-        {'name': 'memory.java.max_heap', 'type': 'gauge'},
         {'name': 'memory.tuple_data', 'type': 'gauge'},
         {'name': 'memory.tuple_allocated', 'type': 'gauge'},
-        {'name': 'memory.tuple_count', 'type': 'gauge'},
         {'name': 'memory.index', 'type': 'gauge'},
         {'name': 'memory.string', 'type': 'gauge'},
+        {'name': 'memory.tuple_count', 'type': 'gauge'},
```

Comment:
Is this just a reordering? Can you leave it as it was?

Reply:
sorry we cant leave it as it was because it resulted in wrong mapping of data from VoltDB causing ambiguity in the charts created

Reply:
Can you test again removing this particular change and tell me exactly which problem it causes?

Reply:
Ok, I have verified on my end and added a test for https://github.com/DataDog/integrations-core/pull/11775 that is currently broken but should get fixed once your change gets in